README.md

```diff
@@ -3,32 +3,27 @@
 
 Welcome to your new Hanami project!
 
-## Setup
+## How to run app inside Docker
```

Comment:
could you move this information to `CONTRIBUTING.md` file without changing readme?

Reply:
No problem :-)

Reply:
Done